README.md

```diff
@@ -1,2 +1,62 @@
-# library
-Javascrip Project
+# Library
+
+## About
+
+This project is a library where the user can add book to his/her collection and mark them as read or not read.
+
+![SCREENSHOT](app_screeshot.png)
```

Comment:
## Fix Screenshot on README
- [ ] The screenshot on this `README` is currently not showing. Do you mind improving it. it should be a screenshot of a section of the page.